src/commands/tag-image.yml

```
@@ -0,0 +1,25 @@
+description: Add a tag to an existing published image
+
+parameters:
+  repo:
+    type: string
+    description: Name of an Amazon ECR repository
+
+  source-tag:
+    type: string
+    description: An existing Docker image tag
+
+  target-tag:
+    type: string
+    description: A comma-separated string containing docker image tags (default = latest)
```

The description indicates this parameter has a default, yet none is defined.

```suggestion
    default: latest
    description: A comma-separated string containing docker image tags (default = latest)
```